deploy-deps.sh

```diff
@@ -48,13 +48,15 @@ deploy_tekton() {
     # Operator
     kubectl apply -k "${script_path}/dependencies/tekton-operator"
     retry kubectl wait --for=condition=Ready -l app=tekton-operator -n tekton-operator pod --timeout=240s
+    # Wait for the operator to create configs for the first time before applying configs
+    kubectl wait --for=condition=Ready TektonConfig/config --timeout=360s
```

Comment:
I tested it with `240s` for several times and it worked.
Do we want to wait this long?

Reply:
Just because that was the number on the original wait line. It might be that @gbenhaim had some tries in which 240 was not enough.